packages/dashboard-backend/src/init.ts

```diff
@@ -0,0 +1,58 @@
+/**********************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import { INodeConfig } from "@eclipse-che/devworkspace-client";
```

Comment:
``` "@eclipse-che/devworkspace-client"``` -> ``` '@eclipse-che/devworkspace-client'````

Reply:
This should be another issue to set up linters and formatters for the whole project to fix such issues.